Sources/Subprocess/AsyncBufferSequence.swift

```diff
@@ -61,7 +59,15 @@ public struct AsyncBufferSequence: AsyncSequence, Sendable {
                 #endif
                 return nil
             }
-            return data
+            let createdBuffers = Buffer.createFrom(data)
```

Comment:
Style nitpick, but this is just `init`.

Reply:
Or, in this case it sort of goes the other way since it vends buffers from data... I guess we can clean it up later.

Reply:
And and also it creates multiple. I didn't use initializer because it's `DispatchData -> [Buffer]`.